tests/load/pipeline/test_bigquery.py

```diff
@@ -0,0 +1,87 @@
+from typing import Iterator, Dict, Union
```

Comment:
you can move these tests to tests/load/biquer

Reply:
Resolved in c9b8c177ba88ae3ed709bbb943a6d2d8dcf239d2